tests/examples/misc_examples.py

```diff
@@ -36,6 +39,8 @@
 
 MOCK_SAVED_QUERY_URL = 'https://mydummyurl.com/saved_queries'
 
+MOCK_LOGAPI_URL = 'https://mydummyurl.com/management'
```

Comment:
and this should end with /logs

I know that its only for unit tests but for consistency better to fix imho.